src/main/java/com/aws/greengrass/deployment/DeviceConfiguration.java

```diff
@@ -349,6 +354,17 @@ public Topic getRootCAFilePath() {
                 .addValidator(deTildeValidator);
     }
 
+    /**
+     * Get JVM options. Default to current JVM options input.
+     *
+     * @return JVM options config topic
+     */
+    public Topic getJvmOptions() {
+        String jvmOptions = ManagementFactory.getRuntimeMXBean().getInputArguments().stream().sorted()
+                .filter(s -> !s.startsWith(JVM_OPTION_ROOT_PATH)).collect(Collectors.joining(" "));
```

Comment:
Hmm why exclude `-Droot`?

Reply:
The loader script will pass in `-Droot` https://github.com/aws-greengrass/aws-greengrass-nucleus/blob/main/scripts/loader#L27 
The Nucleus root path won't be changed even if customers configure it in jvmOptions via deployments